common/src/main/java/io/pravega/common/concurrent/ThreadPoolScheduledExecutorService.java

```diff
@@ -61,6 +62,7 @@
 public class ThreadPoolScheduledExecutorService extends AbstractExecutorService implements ScheduledExecutorService  {
 
     private static final AtomicLong COUNTER = new AtomicLong(0);
+    @Getter
```

Comment:
Set `AccessLevel` to `PACKAGE_PRIVATE`. We don't want to expose this outside of this package.